_docs/tutorials/win-veeam-agent.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Install Veeam Agent for Microsoft Windows
+description: How to install Veeam Agent for Microsoft Windows
+tags: [ featured, coretasks ]
+# permalink: /install-Veeam-Agent-for-Microsoft-Windows/
+---
+# Install Veeam Agent for Microsoft Windows
+{: .no_toc }
+---
+
+In this page, you can find an explanation of how to Install and use Veeam Agent for Microsoft Windows.
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+# Before You Begin
+
+## Prerequisites:
+The computer on which you plan to install Veeam Agent for Microsoft Windows must satisfy [system requirements](https://helpcenter.veeam.com/docs/agentforwindows/userguide/system_requirements.html?ver=40).
+You must run the Veeam Agent for Windows setup file under user account that has Administrator privileges on the computer to install this product.
+Veeam Agent for Windows requires the following components:
+
+> Microsoft .NET Framework 4.7.2
+
+> Microsoft SQL Server 2012 (Local DB)
+
+> Microsoft SQL Server System CLR Types
+
+> Microsoft SQL Server 2012 Management Objects
+
+If these components are not pre-installed on the computer, the setup will install them automatically.
+
+The product program files are placed to the **%ProgramFiles%\Veeam\Endpoint** Backup folder on the system volume. Veeam Agent for Windows requires at least **200 MB**.
+During the recovery image creation, Veeam Agent for Windows formats the removable storage device.
+
+## To install Veeam Agent for Windows
+Save the downloaded archive on the computer where you plan to install the product.
+You should accept the license agreements and click Install.
```

Comment:
you may need to add text selection: 
and click **Install**.

Reply:
Fixed